docs/docker/prestashopCli.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+#http://redsymbol.net/articles/unofficial-bash-strict-mode/
```

Comment:
How about pass PS_BRANCH and PS_VERSION argument instead? This make this script more flexible and easier to maintain